.github/ISSUE_TEMPLATE/coblocks-and-or-gutenberg-update-github-tracking-issue-template.md

```diff
@@ -0,0 +1,39 @@
+---
+name: 'CoBlocks and or Gutenberg update GitHub Tracking Issue template'
+about: 'Template for Gutenberg/Coblocks plugin update process'' GitHub tracking issue,
+  more: '
+title: Update Gutenberg to v<version> and CoBlocks to v<version>
+labels: ''
+assignees: ''
+
+---
+
+We have installed Gutenberg `v<version>` and CoBlocks `v<version>` to edge sites for early feedback. 
+
+Feedback and bugs and the final activation of the plugins on wpcom will be tracked here, for anyone interested.
+
+Testing post: p7DVsv-8Oh-p2
+
+### Process
+
+* [ ] Install version 8.3.0-rc.1. (D44703-code)
+* [ ] Activate version 8.3.0-rc.1. on edge sites (D44704-code)
+* [ ] Install version 8.3.0. (D44796-code)
+* [ ] Activate version 8.3.0 on edge sites. (D44797-code)
+* [ ] Activate 8.3.0 in production for all sites
+     - [ ] Prepare diff (@pingwhoeverisresponsible) (D45045-code)
+     - [ ] Commit/Deploy 
```

Comment:
also might not be helpful to put down diff & deploy (since every step also contains each of these items)